build/AzurePipelinesTemplates/WindowsAppSDK-SourceAnalysis-job.yml

```diff
@@ -0,0 +1,45 @@
+# This job is expected to run in parallel with the build task.
+parameters:
+  - name: buildPool
+    type: string
+    default: 'ProjectReunionESPool'
```

Comment:
This is set as a variable in the pipeline in the other templates.

Should be the case here as well instead of a parameter.

Reply:
Sure, it's now a varaible.